Assets/GothicVR/Scripts/World/DijkstraWaypoint.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace GVR
+{
+    public class DijkstraWaypoint : IComparable<DijkstraWaypoint>
+    {
+        public string _name = "";                                //as index to find other data like position, underwater and probably isFree
+
+        public double _coveredDistance = 0;
```

Comment:
Please follow coding style of C#.

If you use Rider, it will tell you, that underscore is for private (static?) only.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

Reply:
changes made